sema4ai/vscode-client/src/rcc.ts

```diff
@@ -468,7 +486,12 @@ export async function submitIssue(
 
             const metadata = await collectIssueBaseMetadata();
 
+            const currentPackages: any = await langServer.sendRequest("listRobots");
```

Comment:
Given the user is submitting an issue we should have as few dependencies as possible and consider almost everything is in a bad state, so, I think that you should try..catch here for both cases (who knows, the `langServer` may not even be available in case there's an issue starting the action server).